CHANGELOG.md

```diff
@@ -9,6 +9,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+### Added
+
+### Removed
+
+### Changed
+
+### Deprecated
+
+### Security
+
+### Documentation
+
+### Misc
+
+# Past Releases
+
+## [0.16.0] - 2020-09-22
```

Comment:
Can you add this reference link at the bottom, and update the link for main?

Reply:
Ahh right, done!